ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -504,6 +528,30 @@ - (void)mapViewWillStartLoadingMap:(MGLMapView *)mapView
     [_eventDispatcher sendInputEventWithName:@"onStartLoadingMap" body:event];
 }
 
+- (void)offlinePack:(MGLOfflinePack *)pack progressDidChange:(MGLOfflinePackProgress)progress {
+    NSDictionary *userInfo = [NSKeyedUnarchiver unarchiveObjectWithData:pack.context];
+    NSDictionary *event = @{ @"target": self.reactTag,
+                             @"src": @{
+                                     @"name": userInfo[@"name"],
+                                     @"countOfResourcesCompleted": @(progress.countOfResourcesCompleted),
+                                     @"countOfResourcesExpected": @(progress.countOfResourcesExpected),
+                                     @"countOfBytesCompleted": @(progress.countOfBytesCompleted),
+                                     @"maximumResourcesExpected": @(progress.maximumResourcesExpected)
+                                     }
+                             };
+    
+    [_eventDispatcher sendInputEventWithName:@"onProgressDidChange" body:event];
+}
+
+- (void)offlinePack:(MGLOfflinePack *)pack didReceiveMaximumAllowedMapboxTiles:(uint64_t)maximumCount {
+    NSDictionary *event = @{ @"target": self.reactTag,
+                             @"src": @{
+                                     @"maximumCount": @true
```

Comment:
Don't use true in Objective-C; use YES.

But why is this method returning a Boolean anyways? A numeric maximum count should be sent to the client so it knows what the tile limit is.